sui-execution/latest/sui-adapter/src/static_programmable_transactions/typing/invariants/defining_ids_in_types.rs

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{
+    data_store::PackageStore,
+    sp,
+    static_programmable_transactions::{env, spanned::Spanned, typing::ast as T},
+};
+use sui_types::error::ExecutionError;
+
+pub fn verify(env: &env::Env, tt: &T::Transaction) -> Result<(), ExecutionError> {
```

Comment:
I don't actually remember what I did, but let's match the naming scheme for the passes in `verify`